synapse/tools/moduser.py

```diff
@@ -16,24 +17,33 @@ async def main(argv, outp=s_output.stdout):
     pars = argparse.ArgumentParser(prog='moduser', description=descr)
     pars.add_argument('--svcurl', default='cell:///vertex/storage', help='The telepath URL of the Synapse service.')
     pars.add_argument('--add', default=False, action='store_true', help='Add the user if they do not already exist.')
+    pars.add_argument('--del', dest='delete', default=False, action='store_true', help='Delete the user if they exist.')
     pars.add_argument('--admin', choices=('true', 'false'), default=None, help='Set the user admin status.')
     pars.add_argument('--passwd', action='store', type=str, help='A password to set for the user.')
     pars.add_argument('--email', action='store', type=str, help='An email to set for the user.')
     pars.add_argument('--locked', choices=('true', 'false'), default=None, help='Set the user locked status.')
     pars.add_argument('--grant', default=[], action='append', help='A role to grant to the user.')
     pars.add_argument('--revoke', default=[], action='append', help='A role to revoke from the user.')
+    pars.add_argument('--setroles', default=[], action='append',
```

Comment:
Using `--set-roles` and having to specify it multiple times seems inconsistent. Could we make it use `--add-roles` with a comma separated list so we can also add `--del-roles` when someone inevitably requests it?